PocketLobby/constituents
# Password Reset BE

[Trello Ticket](https://trello.com/c/PwgLgsLr)

+ [x] update docs to describe endpoints
+ [x] configure routes to use Devise
+ [x] implement endpoints

## Prerequisite PRs

## Dependent PRs

## New ENV Vars

## Notes for Reviewer
To test it:
- Go to http://localhost:3000/api/password/new
- Add an email that's not in the database. Verify that you get a 404.
- Go back and add an email that is in the database. You should get a 201 and see the reset password link in the logs
- Test curl request
